db/migrate/20250507151848_add_needs_kms_rotation_indices_to_submission_tables.rb

```diff
@@ -0,0 +1,14 @@
+class AddNeedsKmsRotationIndicesToSubmissionTables < ActiveRecord::Migration[7.2]
+  def change
```

Comment:
You need `disable_ddl_transaction!` for adding indexes. https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-database-migrations#vets-apidatabasemigrations-IndexAddition